_scripts/instruction-widget/templates/install/snap.html

```diff
@@ -72,6 +53,11 @@ <h3>Configuring snapd on specific shells</h3>
 
 {{^advanced}}
 {{>installcertbot}}
+  <p>
+  Execute the following instruction on the command line on the machine to ensure
+  that <tt>certbot</tt> command can be run.
+  </p>
+  <pre>sudo ln -s /snap/bin/certbot /usr/sbin/certbot</pre>
```

Comment:
The conversation is a bit confused, but I think we agreed on `/usr/bin/` in the end?

Reply:
Indeed!